ceph-website/build/build

```diff
@@ -3,6 +3,14 @@ set -ex
 
 env
 
+set +e
```

Comment:
`set +e` because I saw script abruptly ends here while running the export command.

Reply:
I don't follow; was export causing an error somehow?

Reply:
yeah, it was an issue with nvm i suppose. I was following this thread which had a couple of suggestions but none of it worked for me here

Reply:
seems pretty hacky, but okay.....

Reply:
https://github.com/nvm-sh/nvm/issues/1985, thought i pasted the issue i followed before but i forgot to